rules/src/main/scala/fix/PackageObjectExport.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 Arktekk
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fix
+
+import scalafix.v1._
+import scala.meta._
+
+class PackageObjectExport extends SemanticRule("PackageObjectExport") {
+  override def fix(implicit doc: SemanticDocument): Patch = {
+    doc.tree.collect { case pkg @ Pkg.Object(_, name, _) =>
```

Comment:
Will this not match package object using the new scala 3 syntax?

Reply:
package objects will be removed in Scala3. 

https://docs.scala-lang.org/scala3/reference/dropped-features/package-objects.html

I believe this should match old package objects using the new syntax.